packages/edit-post/src/components/header/index.js

```diff
@@ -44,16 +49,21 @@ function Header( { setEntitiesSavedStatesCallback } ) {
 
 	const isLargeViewport = useViewportMatch( 'large' );
 
+	const classes = classnames( {
+		'edit-post-header': true,
+		'has-reduced-ui': hasReducedUI,
+	} );
+
 	return (
-		<div className="edit-post-header">
+		<div className={ classes }>
 			<MainDashboardButton.Slot>
 				<FullscreenModeClose />
 			</MainDashboardButton.Slot>
 			<div className="edit-post-header__toolbar">
 				<HeaderToolbar />
 			</div>
 			<div className="edit-post-header__settings">
-				{ ! hasReducedUI && (
+				{
```

Comment:
are these `{` `}` needed here? If I'm not wrong, we could remove them, even the `<>` wrapper. Something like...

```es6
{ ! isPublishSidebarOpened && (
	forcePreviewLink={ isSaving ? null : undefined }
/>
```